src/core/server/saved_objects/types.ts

```diff
@@ -43,26 +43,33 @@ export interface SavedObjectsMigrationVersion {
 }
 
 /**
+ * Don't use this type, it's simply a helper type for {@link SavedObjectAttribute}
```

Comment:
Is it possible to remove the `@public` annotation to assist with this?

Reply:
Yeah that was my first attempt, but then it also gets excluded from our generated docs and in effect a user reading the docs is left with just `export type SavedObjectAttribute = SavedObjectAttributeSingle | SavedObjectAttributeSingle[];` which isn't very useful :(